tests/node.ts

```diff
@@ -169,10 +173,17 @@ async function executeCommands({
         const relayPeerId = await peerIdForIdentity(cmdDef.relayIdentityName)
 
         console.log(`msg: dialing ${cmdDef.targetIdentityName} though relay ${cmdDef.relayIdentityName}`)
-        const { stream } = await node.dialProtocol(
-          new Multiaddr(`/p2p/${relayPeerId}/p2p-circuit/p2p/${targetPeerId.toB58String()}`),
-          TEST_PROTOCOL
-        )
+        const { stream } = await node
+          .dialProtocol(
+            new Multiaddr(`/p2p/${relayPeerId}/p2p-circuit/p2p/${targetPeerId.toB58String()}`),
+            TEST_PROTOCOL
+          )
+          .catch((err) => {
+            console.log(`dialProtocol to ${cmdDef.targetIdentityName} failed`)
```

Comment:
Would rather say `dialling XYZ failed`

Reply:
can be confused with `dial` action above